src/DocumentFormat.OpenXml/OpenXmlPartReader.cs

```diff
@@ -63,28 +66,24 @@ public OpenXmlPartReader(OpenXmlPart openXmlPart) : this()
         /// <param name="openXmlPart">The OpenXmlPart to read.</param>
         /// <param name="readMiscNodes">Specify false to indicate to the reader to skip all miscellaneous nodes. The default value is false.</param>
         public OpenXmlPartReader(OpenXmlPart openXmlPart, bool readMiscNodes)
-            : this(readMiscNodes)
+            : this(openXmlPart, readMiscNodes, ignoreWhitespace: true)
         {
-            if (openXmlPart is null)
-            {
-                throw new ArgumentNullException(nameof(openXmlPart));
-            }
-
-            _xmlReader = CreateReader(openXmlPart.GetStream(FileMode.Open), true, openXmlPart.MaxCharactersInPart, out _standalone, out _encoding);
         }
 
         /// <summary>
         /// Initializes a new instance of the OpenXmlPartReader class using the supplied stream.
         /// </summary>
         /// <param name="partStream">The part stream of the OpenXmlPart to read.</param>
-        public OpenXmlPartReader(Stream partStream) : this()
+        /// <param name="readMiscNodes">Specify false to indicate to the reader to skip all miscellaneous nodes. The default value is false.</param>
+        /// <param name="ignoreWhitespace">Specify true to indicate to the reader to ignore insignificant white space. The default value is true.</param>
+        public OpenXmlPartReader(Stream partStream, bool readMiscNodes = false, bool ignoreWhitespace = true) : this(readMiscNodes)
```

Comment:
again, this needs to be a new constructor

Reply:
Restored the old constructors and added a new one